packages/website/src/examples/layout/Table/TablePredicate.demo.tsx

```diff
@@ -43,6 +44,12 @@ const Demo = () => {
         [table.state.predicates, data],
     );
 
+    const renderSelectOption: TablePredicateProps['renderOption'] = ({option, checked}) => (
```

Comment:
I understand that you needed this in the demo for testing purposes, but adding in the example makes it harder to understand for new devs. I would rather keep the example as it was demonstrating clearly the happy path.

Reply:
I removed the demo for this props